eth_account/account.py

```diff
@@ -367,7 +380,10 @@ def create_with_mnemonic(self,
         return self.from_mnemonic(mnemonic, passphrase, account_path), mnemonic
 
     @combomethod
-    def recover_message(self, signable_message: SignableMessage, vrs=None, signature=None):
+    def recover_message(self,
+                        signable_message: SignableMessage,
+                        vrs: Optional[Tuple[VRS, VRS, VRS]] = None,
+                        signature: bytes = None) -> Any:  # i/o ChecksumAddress due to decorator.
```

Comment:
Instead of leaving this as `Any`, we can change it to be `ChecksumAddress`, and then change line 474 to be a cast:

`return cast(ChecksumAddress, self._recover_hash(message_hash, vrs, signature)`